mountpoint-s3/src/main.rs

```diff
@@ -340,6 +340,13 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
         .transpose()
         .context("Failed to parse endpoint URL")?;
 
+    let ec2_instance_type = retrieve_instance_type().unwrap_or_else(|_| {
+        tracing::error!("Unable to detect instance type.");
```

Comment:
Need to log the actual error we got back

Reply:
Changed.